torch/quantization/_numeric_suite.py

```diff
@@ -437,7 +422,7 @@ def compare_model_outputs(
     q_model,
     *data,
     Logger=OutputLogger,
-    white_list=DEFAULT_NUMERIC_SUITE_COMPARE_MODEL_OUTPUT_WHITE_LIST,
+    white_list=None
```

Comment:
Should we change the `white_list` while we are at it?

Reply:
i'm afraid this is bc breaking, would be better to change all at once